addons/bus/models/assetsbundle.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from odoo.addons.base.models.assetsbundle import AssetsBundle
+from odoo.http import request
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class BusAssetsBundle(AssetsBundle):
+    TRACKED_BUNDLES = ['web.assets_common', 'web.assets_backend']
+
+    def save_attachment(self, type, content):
+        """
+        Each time an attachment is saved
+        Send a bus notification
+        An attachment is saved when its hash has changed
+
+        @override
+        """
+        saved = super().save_attachment(type, content)
+
+        if self.name in self.TRACKED_BUNDLES:
```

Comment:
Just to be sure.
```suggestion
        if request and self.name in self.TRACKED_BUNDLES:
```

Reply:
I don't need the request anymore, since I don't need to enforce that the same request that computed the bundles flushes the messages.
I do need an env though